contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/__init__.py

```diff
@@ -157,4 +161,9 @@ def new_metrics_exporter(**options):
                                     producers,
                                     exporter,
                                     interval=exporter.options.export_interval)
+    from opencensus.ext.azure.metrics_exporter import statsbeat_metrics
```

Comment:
Imports are usually at the top of the file, any reason is in here?

Reply:
No big reason. Was just easier at the time to see the import.

Reply:
group import together is a good practice.